runtime/doc/tutor.txt

```diff
@@ -0,0 +1,33 @@
+*tutor.txt* Interactive tutorials.
+Author: Felipe Morales <hel.sheep@gmail.com>
+
+*vim-tutor-mode*
+
+                                VIM-TUTOR-MODE
+
+vim-tutor-mode provides a system to create and follow interactive tutorials
+for vim and third party plugins. It replaces the venerable `vimtutor` system.
+
+- USAGE                                                      *vim-tutor-usage*
+
+vim-tutor-mode tutorials are hypertext documents, that is, they have rich text
+and contain links. To stand out from the rest of the text, links are
+underlined. You can follow them by placing the cursor over them and pressing
+<Enter>, or by double-clicking them. Most of the links will open vim's help
+system, but some will move you around the document itself.
+
+- COMMANDS                                                  *:Tutor*
+
+:Tutor {tutorial}           Opens a tutorial.
+			    If no arguments are provided, it opens the
+			    '01-vim-beginner' tutorial, which is equivalent to
+			    Vim's vimtutor.
+
+- CREATING                                                  *vim-tutor-create*
+
+Writing vim-tutor-mode tutorials is easy. Please consult the tutor.tutor
+file: >
+
+    :Tutor tutor
+<
+" vim: set ft=help :
```

Comment:
Should be `pi_tutor.txt` I think, especially since it doesn't follow the documentation style in the nvim docs.